### PR TITLE
Alter the default empty string in json according to the default response

### PR DESCRIPTION
**Pull request by toshetah**

### CODE REVIEW

File: azure-kusto-data/tests/test_kusto_client.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def test_sanity_query(self, mock_post):
             "xnumberAsText": text_type(""),
             "xtime": None,
             "xtextWithNulls": text_type(""),
-            "xdynamicWithNulls": None,
```
Comment (yifatso) [inline azure-kusto-data/tests/test_kusto_client.py]: why did you remove the null checks? (IIUC). can you test null dynamic values (not empty strings)?

Comment (toshetah) [reply azure-kusto-data/tests/test_kusto_client.py]: I am testing it at [test_dynamic](https://github.com/Azure/azure-kusto-python/blob/2aa575f1442c307ff5d7e4f021aa9d1760f1cb8b/azure-kusto-data/tests/test_kusto_client.py#L350) at test_kusto_client.py.
